### ISSUE TITLE
NTR: Carnitine-acylcarnitine translocase low in fibroblasts (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8163]
New term request Carnitine-acylcarnitine translocase low in fibroblasts (f432): CACT

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000574
- label: Reduced tissue carnitine-acylcarnitine translocase activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:9323572
- def: Activity of carnitine-acylcarnitine translocase (CACT) in tissues below the lower limit of normal. CACT activity can be measured in multiple tissues.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8163
- synonyms:
